src/http.js

```diff
@@ -1,11 +1,111 @@
+/* eslint-disable prefer-destructuring */
+const Airtable = require("airtable");
+
 const express = require("express");
-const { logger } = require("./logger/");
+const bodyParser = require("body-parser");
+const config = require("./config");
+const { logger } = require("./logger");
+const { slackConf } = require("./slack/index");
+const { getButtons, handleButtonUpdate } = require("./slack/dispatch");
 
+const reminderObj = { id: null, date: null, time: null };
+const reminderArray = [];
```

Comment:
Seems to me that both `reminderArray` and `reminderObj` shouldn't be stored as global variables. Cases to consider:
- What is the bot restarts and `reminderArray` is reset to `[]`?
- What if multiple requests come into `/slack/actions/` at the same time?

Reply:
reminderArray exists so multiple people can submit reminders at the same time. The app restarting and reminderArray being set to [] would only affect reminders that had been started but not completed. As in they click "Flag for Followup" but dont select a date or time and just forget or keep working on something else. The app would keep this info in the array and would still be able to be set until the app restarts

Reply:
If this isn't ideal, I can look into finding a better way to store this data

Reply:
Yeah, expect that the app can restart. Just store data in a field in Airtable?

Reply:
Removed reminderObj and reminderArray, this data is now being saved in the reminder date/time field in airtable. Once a user selects a date for their reminder, the reminder is set for 8am of that day until they select a reminder time, at which point the reminder date/time is updated to reflect their selection. All reminder data now survives a reboot